OWNERS

```diff
@@ -4,10 +4,11 @@ approvers:
   - cyriltovena
   - dzlier-gcp
   - EricFortin
-  - ilkercelikyilmaz
-  - jkowalski
-  - joeholley
   - markmandel
   - pooneh-m
   - roberthbailey
 
+emeritus_approvers:
```

Comment:
Does this mean this group cannot approve anymore or will not be suggested as reviewer?

Reply:
Yes (to both). See https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus.

Reply:
Thanks. Does it make sense that the same set of people who their role is being changed to emeritus_approvers, approve the change or if not responding then someone else approve? 

The same [link](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus) says:

>  **They** are encouraged to add **themselves** as an "emeritus" approver under the emeritus_approvers key.